project/cmake/CMakeLists.txt

```diff
@@ -117,7 +117,7 @@ endif()
 
 # Optional dependencies
 set(optional_deps MicroHttpd MySqlClient SSH XSLT
-                  Alsa UDEV DBus Avahi
+                  Alsa UDEV DBus Avahi SmbClient
```

Comment:
Right - to the bettet from the looks of it.

-------- Opprinnelig melding --------
Fra: Wolfgang Schupp <notifications@github.com>
Dato: 30.08.2016 13:09 (GMT+01:00)
Til: xbmc/xbmc <xbmc@noreply.github.com>
Ko: Arne Morten Kvarving <Arne.Morten.Kvarving@sintef.no>, Comment <comment@noreply.github.com>
Emne: Re: [xbmc/xbmc] Linux multi arch (#10359)


In project/cmake/CMakeLists.txt<https://github.com/xbmc/xbmc/pull/10359#discussion_r76774259>:

> @@ -117,7 +117,7 @@ endif()
>
>  # Optional dependencies
>  set(optional_deps MicroHttpd MySqlClient SSH XSLT
> -                  Alsa UDEV DBus Avahi
> +                  Alsa UDEV DBus Avahi SmbClient


enable switch for optional deps was changed in fe50635<https://github.com/xbmc/xbmc/commit/fe50635fb7a09d1576890a5d548429d55af00221>

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub<https://github.com/xbmc/xbmc/pull/10359/files/9a760f8a89728f7e3c061ccd18b690bccb8bb9ed..9d33e3d8bbddc6b891f3de8f30f8d14499cd88be#r76774259>, or mute the thread<https://github.com/notifications/unsubscribe-auth/ACC4RwZqU9_NZLba4Y6ONveXVkBxnTXkks5qlA8fgaJpZM4JveFV>.